ci/docker/init.sh

```diff
@@ -9,6 +9,7 @@ docker-compose up -d
 docker-compose exec -T php-fpm.stepup.example.com bash -c '
   cp ./ci/config/*.yml ./app/config/
```

Comment:
Please remove the `cp` actions, only `composer up` should be sufficient to get this working to prevent configuration all over the place This script shouldn't manage state and this should be moved to service and parameter configuration. I've fixed this earlier on. It would even be better to even move the `composer` commands but this won't affect state so it could be left here to bootstrap the box. Ultimately this should be moved to the docker file itself.